src/Language/Printer.php

```diff
@@ -226,6 +228,7 @@ public function printAST($ast)
                     },
 
                     NodeKind::NAMED_TYPE => static function (NamedTypeNode $node) : string {
+                        // @phpstan-ignore-next-line the printer works bottom up, so this is already a string here
```

Comment:
assert(is_string()) might be more suitable then?

Reply:
There is nothing external that might mess this up, we are not calling user code in the process of schema printing. It is just that the AST is manipulated in ways that are unusual and cause it to deviate from its usually well-typed form.

That modification is contained solely within this class and is also completely deterministic, so I think an `assert()` is out of place.

Reply:
It would be to satisfy **only that one particular** phpstan issue. It's more specific than ignoring **all** errors on the next line.

Reply:
Agreed, i added `assert()` instead.